src/pages/CategoriesPage/Table/CategoriesTableComponent.tsx

```diff
@@ -1,17 +1,17 @@
-import { DataGrid } from '@mui/x-data-grid';
-import { categoryTableColumns } from '../../../constants/categoryTableColumns';
+import { DataGrid, GridColDef } from '@mui/x-data-grid';
 import { CategoryModel } from '../../../models/categories/CategoryModel';
 import { styles } from '../styles';
 
 interface Props {
   categories: CategoryModel[];
+  categoryTableColumns: GridColDef[]
 }
 
 const CategoriesTableComponent = (props: Props) => {
   return (
     <div style={styles.tableWrapper}>
```

Comment:
Do not use DIV. Use Box instead.

Reply:
All divs in the modal windows have been replaced by the Box components